packages/fx-core/src/plugins/resource/frontend/plugin.ts

```diff
@@ -320,4 +331,31 @@ export class FrontendPluginImpl {
     );
     await EnvironmentUtils.writeEnvironments(envFilePath, envs);
   }
+
+  public async executeUserTask(func: Func, ctx: PluginContext): Promise<TeamsFxResult> {
+    if (func.method === "migrateV1Project") {
```

Comment:
Nit: I think @hund030 would like make the string an enum, but let he decides the definition location.